clang/lib/DPCT/APINames_removed.inc

```diff
@@ -40,3 +40,4 @@ ENTRY(cuMemHostUnregister,             "SYCL currently does not support register
 ENTRY(cudaProfilerStart,               "SYCL currently does not support this function. Remove the profiler API will not impact the outcome.")
 ENTRY(cudaProfilerStop,                "SYCL currently does not support this function. Remove the profiler API will not impact the outcome.")
 ENTRY(cuFuncSetAttribute,              "SYCL currently does not support setting kernel function attributes")
+ENTRY(cuGetExportTable,                "the functionality of the retrieved table is undocumented.")
```

Comment:
```suggestion
ENTRY(cuGetExportTable,                "the functionality is not required.")
```

Reply:
The message is updated to be more precise.